### PR TITLE
Add the ability to override the cc address

### PR DESCRIPTION
**Pull request by andrew-edgar**
Changed to fix the override ability of the cc address.

### CODE REVIEW

**Comment by Benjamintf1** [comment]
This might take a bit to investigate. PR pipeline seems to have trouble with branches with the same name 😨. It's *somewhat* new for me to use this way, so I'm going to try and get it to work. Might take a bit tho.

**Comment by Benjamintf1** [comment]
They're actually running now.

**Comment by Benjamintf1** [comment]
@andrew-edgar I'm taking a little bit of a look, but this is the error I'm seeing right now
image

File: jobs/reverse_log_proxy/templates/bpm.yml.erb [hunk 1]
```diff
@@ -1,13 +1,13 @@
 <%
   ingress_addrs = []
-  if_link("doppler") { |ds|
-      ingress_addrs = ds.instances.map do |instance|
-          "#{instance.address}:#{ds.p('doppler.grpc_port')}"
-      end
-  }.else {
+  if_p("loggregator.doppler.addrs") { |daddrs|
```
Comment (Benjamintf1) [inline jobs/reverse_log_proxy/templates/bpm.yml.erb:3]: The order changed on the list, but loggregator.doppler.addrs has a default of '[]' unlike TC which has no default. Please change the default this property to not exist.

**Review by Benjamintf1 — CHANGES_REQUESTED** [review]

**Comment by andrew-edgar** [comment]
That default for doppler.addrs was not my code but I removed the default as requested

**Comment by Benjamintf1** [comment]
image

This is the failure.

**Comment by Benjamintf1** [comment]
you're going to want to make that change on *all* the jobs, namely rlp(the currently failing job).

**Review by Benjamintf1 — APPROVED** [review]

**Comment by Benjamintf1** [comment]
@andrew-edgar https://github.com/cloudfoundry/loggregator-release/releases/tag/v106.6.0

**Pull request merged**